### DISCUSSION TITLE
pulizia di pasqua

### DISCUSSION THREAD

**Discussion by RtoRoc** [indomus/forum#32] [HUB: Home Assistant]
salve. ho notato che se vado su inpostazioni/entita mi ritrovo delle entita di alcune automazioni che in realta ho cancellato da svariati mesi.... mi e stato suggerito di fare una pulizia della cartella storage..., ho cercato on line ma l'unica cosa che ho trovato e la guida indomus su come effettuare questa operazione con installazione home assistant su raspbian,,,,, ma non ho trovato niente in relazione ad hassio.... qualcuno sa aiutarmi?

**Comment by snakuzzo** [comment]
Andare a manella sulla directory `.storage` te lo sconsiglio...soprattutto se non sai cosa stai facendo.
Piuttosto vai sulla webgui, sotto  `CONFIGURATION > ENTITIES`, ordina le entità per stato e vedrai che ti segnerà in rosso quelle che potrai eliminare

**Reply by RtoRoc** [reply]
> Andare a manella sulla directory te lo sconsiglio... soprattutto se non sai cosa stai facendo.
> Piuttosto vai sul webgui, sotto , ordina le entità per stato e vedrai che ti segnerà in rosso quelle che potrai eliminare`.storage``CONFIGURATION > ENTITIES`

si si quello ok..... il fatto e che ho un botto di entita non disponibili tra le quali tutti i vari switch degli amazon echo che  non servono a un cazzo e in fatti ho disabilitato e poi ho una cifra di automaznioni che ho eliminato tipo 3 mesi fa ma che sono ancora li a rompere le palle non si sa a fare cosa....... concludo  con una parola che ti piace tanto PACKAGES
